includes/server-timing/class-perflab-server-timing.php

```diff
@@ -25,7 +25,10 @@ class Perflab_Server_Timing {
 	 * Map of registered metric slugs and their registered data.
 	 *
 	 * @since 1.8.0
-	 * @var array<string, array{measure_callback: callable, access_cap: string}>
+	 * @var array<string, array{
```

Comment:
FYI to avoid repetition you could also use `@phpstan-type` in the class doc block and re-use that same type everywhere else in the class.

See https://phpstan.org/writing-php-code/phpdoc-types#local-type-aliases

Reply:
Good idea! Done in 64458ab2182f2abafa6e93b034501644c51dd1ef.